lib/v_0.7.0/init.js

```diff
@@ -48,7 +48,7 @@ module.exports = function(appName, options) {
    * doing it this way sets fugitivelabs/yote.git as the upstream master and
    * allows the user to pull from the upstream remote
    */
-  shell.exec("git clone https://github.com/fugitivelabs/yote.git " + appName);
+  shell.exec("git clone -b v0.7-release --single-branch https://github.com/fugitivelabs/yote.git " + appName);
```

Comment:
We should prob change this back once 0.8 Yote is merged to master

Reply:
This one should actually always hit 0.7, its the 0.8/init.js one that should hit master. We want this one to preserve functionality for our older clients.